lib/griddler/sendgrid/adapter.rb

```diff
@@ -48,7 +48,7 @@ def remove_addresses_from_bcc(bcc, addresses)
       end
 
       def bcc_from_envelope(envelope)
-        JSON.parse(envelope)["to"] if envelope
+        JSON.parse(envelope)["to"] if !envelope.blank?
```

Comment:
Personally I prefer `foo.present?` over `!foo.blank?`... take it or leave it, your option :grin: